portfolio/src/main/webapp/index.html

```diff
@@ -1,24 +1,273 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="UTF-8">
-    <title>Dan Abreo</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>Dan's Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>
-        I am an incoming Junior at Texas A&M University, pursuing a Bachelor's Degree in Computer Science.
-        I was born and raised in Houston, Texas near NASA's Johnson Space Center, 
-        and I enjoy putting my software engineering skills to use in solving real-world problems.
-      </p>
-      <img style="width:100%" src="/images/nasa.jpg" alt="Photo of Dan in the Space Shuttle"></img> 
-      <p>Click here to get a random fact about me:</p>
-      <button onclick="addRandomFact()">Get a fact!</button>
-      <div id="fact-container"></div>
-    </div>
-  </body>
-</html>
+<!doctype html>
```

Comment:
nit: change `doctype` to uppercase [1]. 

[1] https://html.spec.whatwg.org/multipage/syntax.html#the-doctype